bigquery_etl/operational_monitoring/templates/scalar_query.sql

```diff
@@ -110,8 +110,9 @@ SELECT
     submission_date,
     client_id,
     build_id,
-    cores_count,
-    os,
+    {% for dimension in dimensions %}
+      {{ dimension.name }},
+    {% endfor %}
```

Comment:
instead of explicitly listing the dimension here, only use the ones that are actually in the config